electrum/transaction.py

```diff
@@ -88,7 +88,16 @@ class MissingTxInputAmount(Exception):
     pass
 
 
-SIGHASH_ALL = 1
+class Sighash(IntFlag):
+    SIGHASH_ALL = 1
+    SIGHASH_NONE = 2
+    SIGHASH_SINGLE = 3
+    SIGHASH_ANYONECANPAY = 0X80
```

Comment:
I think we should just remove the `SIGHASH_` prefixes, as `Sighash.ALL` is clear enough (no need for `Sighash.SIGHASH_ALL`).

Reply:
Please use lowercase `x` for hex literals: `0x80`.